app/main.py

```diff
@@ -3,17 +3,19 @@ def __init__(
             self,
             comfort_class: int,
             clean_mark: int,
-            brand: str) -> None:
+            brand: str
+    ) -> None:
         self.comfort_class = comfort_class
         self.clean_mark = clean_mark
         self.brand = brand
 
 
 class CarWashStation:
-    def __init__(self,
-                 distance_from_city_senter: float,
-                 clean_power: int, average_rating: float,
-                 count_of_ratings: int) -> None:
+    def __init__(
+            self,
+            distance_from_city_senter: float,
+            clean_power: int, average_rating: float,
+            count_of_ratings: int) -> None:
```

Comment:
`) -> None:` should end on the new line. And you also have 2 arguments on line 17. Start every argument from a new line

```
def __init__(
        self,
        distance_from_city_senter: float,
        clean_power: int, average_rating: float,
        count_of_ratings: int
) -> None:
```